salesapp.js

```diff
@@ -1,16 +1,38 @@
 'use strict';
 
-function CookieStore(idTag, storeName, minHourCustomers, maxHourCustomers, avgCookiesPerCustomer){
-  this.idTag = idTag;
+//-------------------------CONSTRUCTOR OBJECT-------------------
+function CookieStore(storeName, minHourCustomers, maxHourCustomers, avgCookiesPerCustomer){
   this.storeName = storeName;
   this.minHourCustomers = minHourCustomers;
   this.maxHourCustomers = maxHourCustomers;
   this.avgCookiesPerCustomer = avgCookiesPerCustomer;
   this.hourlyCookies = [];
   this.totalSales = 0;
 };
+//-------------------------------FORM----------------------------
+var storeFormEl = document.getElementById('new-store-form');
+var formStore;
+storeFormEl.addEventListener('submit', handleSubmit, false);
+var formStore;
+function handleSubmit(event){
+    // var formStore = new CookieStore(formNewStoreName, formMinCustomers, formMaxCustomers, formAvgCookies);
+  event.preventDefault();
+  event.stopPropagation(); //stops all bubbling/capturing
+  var formNewStoreName = event.target.cookieStoreName.value;
+  var formMinCustomers = parseInt(event.target.minCust.value);
+  var formMaxCustomers = parseInt(event.target.maxCust.value);
+  var formAvgCookies = parseInt(event.target.avgCust.value);
 
+   //VERY IMPORTANT! This function pulls the data in the selected field.
+  //event = whole element, target = storeFormEl, cookieStoreName = the specific input node, value = input inside of text field.
+  // console.log(formNewStoreName, formMinCustomers, formMaxCustomers, formAvgCookies);
+  formStore = new CookieStore(formNewStoreName, formMinCustomers, formMaxCustomers, formAvgCookies);
+  console.log(formStore);
+  console.log('User pressed submit button on form');
 
+
+
+//-------------------------------METHODS---------------------------------
 CookieStore.prototype.getAvgCookieCount = function() {
```

Comment:
Group the functions that you attach to the prototype of a constructor up with the constructor itself. That way you'll be able to find things.